package.json

```diff
@@ -68,6 +71,7 @@
     "leaflet": "^1.7.1",
     "lodash.debounce": "^4.0.8",
     "lodash.isequal": "^4.5.0",
+    "material-ui-color-picker": "^3.5.1",
```

Comment:
ok

Reply:
Ran npx depcheck and got a list

I'm removing these from package.json (some will still be pulled in by yarn.lock due to dependencies)
-------
* @codemirror/language-data
* @nivo/core
* material-ui-color-picker
* react-router

I'm keeping these (because depcheck isn't 100% accurate. The ones I am removing above I checked the code)
-------
* babel-runtime
* lodash.debounce
* postcss
* postcss-loader
* postcss-preset-env